var/da/da_main/da_update_firstguess.inc

```diff
@@ -16,7 +16,8 @@ subroutine da_update_firstguess(grid, out_filename)
 !    grid%w_2
 !    grid%mu_2
 !    grid%ph_2
-!    grid%t_2
+!    grid%t_2 (dry potential temperature perturbation)
```

Comment:
@liujake 
Jake,
In the WRF model, grid%t_2 is either moist or dry theta perturbation, depending on the use_theta_m switch. The grid%th_phy_m_t0 is always the dry theta perturbation. The THM name is a character string of the input / output field for the grid%t_2 variable. 

Inside of the DA Registry, grid%t_2 is always the dry theta perturbation.

For this comment, we should be careful about referring to the internal field (such as grid%t_2) and the string that is used to name the field in the input / output file.

Reply:
@davegill I added your text in the subroutine as a note to clarify this.